pmm/pmm2-ovf-staging-start.groovy

```diff
@@ -105,5 +139,83 @@ pipeline {
+                        sh """
+                            ssh -i "${KEY_PATH}" -p 3022 -o ConnectTimeout=1 -o StrictHostKeyChecking=no admin@${PUBLIC_IP} '
+                                sudo yum update -y percona-release || true
+                                sudo sed -i'' -e 's^/release/^/experimental/^' /etc/yum.repos.d/pmm2-server.repo
+                                sudo percona-release enable percona experimental
+                                sudo yum clean all
+                            '
+                        """
+                    }
+                }
+            }
+        }
+        stage('Enable Release Repo') {
+            when {
+                expression { env.ENABLE_EXPERIMENTAL_REPO == "no" && env.ENABLE_TESTING_REPO == "no" }
+            }
+            steps {
+                node(env.VM_NAME) {
+                    withCredentials([sshUserPrivateKey(credentialsId: 'OVF_VM_TESTQA', keyFileVariable: 'KEY_PATH', passphraseVariable: '', usernameVariable: 'USER')]) {
+                        sh """
+                            ssh -i "${KEY_PATH}" -p 3022 -o ConnectTimeout=1 -o StrictHostKeyChecking=no admin@${PUBLIC_IP} '
+                                sudo yum update -y percona-release || true
+                                sudo yum clean all
+                            '
+                        """
+                    }
+                }
+            }
+        }
+        stage('Setup QA Repo on OVF VM') {
+            steps {
+                node(env.VM_NAME) {
+                    withCredentials([sshUserPrivateKey(credentialsId: 'OVF_VM_TESTQA', keyFileVariable: 'KEY_PATH', passphraseVariable: '', usernameVariable: 'USER')]) {
+                        sh """
+                            ssh -i "${KEY_PATH}" -p 3022 -o ConnectTimeout=1 -o StrictHostKeyChecking=no admin@${PUBLIC_IP} '
+                                export PMM_QA_GIT_BRANCH=${PMM_QA_GIT_BRANCH}
+                                export PMM_QA_GIT_COMMIT_HASH=${PMM_QA_GIT_COMMIT_HASH}
+                                sudo yum install -y wget
+                                sudo mkdir -p /srv/pmm-qa || :
+                                pushd /srv/pmm-qa
+                                    sudo git clone --single-branch --branch \${PMM_QA_GIT_BRANCH} https://github.com/percona/pmm-qa.git .
+                                    sudo git checkout \${PMM_QA_GIT_COMMIT_HASH}
+                                    sudo wget https://raw.githubusercontent.com/Percona-QA/percona-qa/master/get_download_link.sh
+                                    sudo chmod 755 get_download_link.sh
+                                popd
+                                sudo chmod 755 /srv/pmm-qa/pmm-tests/pmm-framework.sh
+                            '
+                        """
+                    }
+                }
+            }
+        }
     }
 }
```

Comment:
We'll need to add this something like this:
```
            script {
                def node = Jenkins.instance.getNode(env.VM_NAME)
                Jenkins.instance.removeNode(node)
            }
```
We need to remove OVF node from Jenkins after we stopped the instance.
Here example:

https://github.com/Percona-Lab/jenkins-pipelines/blob/master/pmm/aws-staging-start.groovy#L483-L488

(I forget about it)

Reply:
Done, thank you